### PR TITLE
[Repo] Corvax

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Changelog

:cl: DEATHB4DEFEAT, courtesy of Corvax
- tweak: Typing indicators are more reactive, with a new state for idling in the chat bar
- tweak: Resprite typing indicators
- add: Slimepeople, Moths, Aliens, and Robots now have unique typing indicators
- add: Added a queue system, if the server is full, players will be put in a queue instead of being kicked
- add: Added random station goals, sent to the command fax machine at the start of the shift
- add: Added some new lobby art
- add: Added more Captain clothing
- add: Added a HoP coat
- add: Added 9 potted plant variations
- tweak: Different shotguns have different sounds
- tweak: Resprite ERT jumpsuits and helmets
- tweak: Resprite Syndicate EVA suit
- tweak: Resprite Singularity and related equipment
- tweak: Resprite intercom
- tweak: Resprite lamps
- tweak: Resprite reagent tanks
- fix: Resprite uranium windows
- tweak: Resprite chairs
- tweak: Resprite mailing units
- tweak: Resprite diagonal solid walls
- tweak: Resprite Atmospheric Technician backpacks
- tweak: Resprite cans
- tweak: Resprite gas canisters
- tweak: Resprite fax machines
- tweak: Resprite some melee weapons

### CODE REVIEW

**Comment by Pspritechologist** [comment]
We need to sort out the indicators, customize the station goal system and write up some of our own, and double-triple sanity check that everything's working fine.

**Comment by DEATHB4DEFEAT** [comment]
maybe take some stuff from these too

https://github.com/space-syndicate/space-station-14/pulls?q=is%3Apr+%5Bresprite%5D+
https://github.com/space-syndicate/space-station-14/pull/854
https://github.com/space-syndicate/space-station-14/pull/844

**Comment by Pspritechologist** [comment]
We can just go with the Corvax indicators. It's easy to revert, not that big of a deal, and honestly speaking I get enough things my way anyway :P
I did say before that I'd give them a go and see if I can get used to them.

File: Resources/Prototypes/typing_indicator.yml [hunk 1]
```diff
@@ -1,27 +1,63 @@
...
+  typingState: blob0
+  idleState: blob3
+
+- type: typingIndicator
+  id: alienroyal
+  typingState: alienroyal0
+  idleState: alienroyal3
+# Parkstation-TypingIndicators-End
+
+# TODO: need sprite for this maybe
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/typing_indicator.yml]: need to resprite moff typing indicator

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**